test/mini_racer_test.rb

```diff
@@ -539,38 +545,42 @@ def test_empty_isolate_is_valid_and_can_be_GCed
     GC.start
   end
 
-  def test_isolates_from_snapshot_dont_get_corrupted_if_the_snapshot_gets_warmed_up_or_GCed
-    # basically tests that isolates get their own copy of the snapshot and don't
-    # get corrupted if the snapshot is subsequently warmed up
-    snapshot_source = <<-JS
-      function f() { return Math.sin(1); }
-      var a = 5;
-    JS
+# segfault
+# CFUNC  :init_with_snapshot
+# METHOD /Users/loic.nageleisen/Source/github.com/rubyjs/libv8-node/test/mini_racer/lib/mini_racer.rb:81 (Isolate#initialize)
```

Comment:
Initialising the isolate with a snapshot burns into flames. Note that it apparently doesn't reach `warmup!` so probably either unrelated to the above or indirectly related but for yet unknown, deeper reasons.